README.md

```diff
@@ -0,0 +1,75 @@
+# AtomicLevels
+
+[![Documentation][docs-stable-img]][docs-stable-img]
+[![Documentation (dev)][docs-dev-img]][docs-dev-img]
+[![GitHub Actions CI][ci-gha-img]][ci-gha-url]
+[![CodeCov][codecov-img]][codecov-url]
+
+AtomicLevels provides a collections of types and methods to facilitate working with atomic states (or, more generally, states with spherical symmetry), both in the relativistic (eigenstates of `J = L + S`) and non-relativistic (eigenstates on `L` and `S` separately) frameworks.
+
+
+* Orbitals and orbital subshells
+* Configurations
+* Configuration state functions (CSFs)
+* Term symbols
+
+The aim is to make sure that the types used to label and store information about atomic states are both efficient and user-friendly at the same time.
+In addition, it also provides various utility methods, such as generation of a list CSFs corresponding to a given configuration, serialization of orbitals and configurations, methods for introspecting physical quantities etc.
+
+## Example
+
+To install and load the package, you can just type in the Julia REPL
+
+```julia-repl
+julia> using Pkg; Pkg.add("DataFrames")
+julia> using AtomicLevels
+```
+
+As a simple usage example, constructing a configuration for an S-like state with an open `3p` shell looks like
+
+```julia-repl
+julia> configuration = c"[Ne]* 3s2 3p4"
+1s² 2s² 2p⁶ 3s² 3p⁴
+```
+
+which is of type `Configuration`. To access information about subshells, you can index into
+the configuration which returns a tuple. Tuple contains an `Orbital` object, so you
```

Comment:
"Tuple contains an..." is missing an article, i.e. "The tuple contains an..."

Reply:
```suggestion
the configuration which returns a tuple. The tuple contains an `Orbital` object, so you
```